src/Plugin/Commerce/PaymentGateway/HostedPaymentFields.php

```diff
@@ -174,6 +178,9 @@ public function createPayment(PaymentInterface $payment, $capture = TRUE) {
       'currency' => $amount->getCurrencyCode(),
       'amount' => $amount->getNumber(),
       'cardTransactionType' => $capture ? 'AUTH_CAPTURE' : 'AUTH_ONLY',
+      'transactionFraudInfo' => [
+         "fraudSessionId" => $this->fraudSession->get(),
+      ],
```

Comment:
**CHANGE REQUEST**

*Readability*

Let's be consistent and use single quotes (`'`) like everywhere else.

Reply:
Done